README.md

```diff
@@ -624,4 +624,49 @@ $ git commit
 
 ```
 
+## GraphQL guideline
 
+### Relay connection
+
+When writing GraphQL query that includes Relay connection type, make sure to include `@connection` directive with `key` set to name of the connection (see example). It's used for better cache normalisation in ApolloClient.
+
+Example:
+
+```
+query watchlistQuery($id: ID!, $first: Int!) {
+    ...
+    watchlistItemConnection(first: $first) @connection(key: "watchlistItemConnection") {
+        edges {
+            node {...}
+        }
+    }
+}
+```
+
```

Comment:
Pls add a simple explanation, why we should add `@connection` and an example implementation of `cache delete`